transformations/random_deletion/README.md

```diff
@@ -0,0 +1,33 @@
+# Butter Fingers Perturbation 🦎  + ⌨️ → 🐍
+This perturbation randomly remove each word of a sentence or paragraph with a probability p. 
+
+Author name: Emile Chapuis and Pierre Colombo 
+Author email: chapuis.emile@gmail.com and colombo.pierre@gmail.com
+Author Affiliation: Telecom Paris - IPP
+
+## What type of a transformation is this?
+This transformation acts like a perturbation to test robustness. Each word of the given text are randomly deleted. 
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks which have a sentence/paragraph/document as input like text classification, 
+text generation, etc. 
+
+## Previous Work
+1) There has also been some recent work in the contrast sets of the GEM Benchmark (ACL 2021):
+```bibtex
+@article{DBLP:journals/corr/abs-1901-11196,
```

Comment:
Might want to cite the paper on word dropout too : https://arxiv.org/pdf/1511.01432.pdf

Reply:
And maybe this too: https://books.google.co.in/books?id=8wklDwAAQBAJ&pg=PT154&lpg=PT154&dq=%22word+dropout+may+also+be+beneficial+for+preventing+overfitting+and+improving+robustness+by+not+letting+the+model+rely+too+much+on+any+single+word+being+present&source=bl&ots=wFYULSyepA&sig=ACfU3U0YiewFhO-XZVDNt7caKclGUmdqSQ&hl=en&sa=X&ved=2ahUKEwij_aef5YPyAhVpIbcAHX1HAE4Q6AEwDnoECBcQAw#v=onepage&q=%22word%20dropout%20may%20also%20be%20beneficial%20for%20preventing%20overfitting%20and%20improving%20robustness%20by%20not%20letting%20the%20model%20rely%20too%20much%20on%20any%20single%20word%20being%20present&f=false